backend/src/main/java/com/google/sps/notifs/EmailNotifier.java

```diff
@@ -128,33 +77,93 @@ private static Message createMessageWithEmail(MimeMessage emailContent)
    * @return the MimeMessage to be used to send email
    * @throws MessagingException if there was a problem accessing the Store
    */
-  private MimeMessage createEmail(String subject, String bodyText) throws MessagingException {
+  private MimeMessage createEmailWithSingleRecipient(
+      String recipientUsername, String subject, String bodyText) throws MessagingException {
 
     Properties props = new Properties();
     Session session = Session.getDefaultInstance(props, /* authenticator= */ null);
 
     MimeMessage email = new MimeMessage(session);
 
     email.setFrom(new InternetAddress(APPLICATION_EMAIL));
-    email.addRecipient(RecipientType.TO, new InternetAddress(recipientEmail));
+    email.addRecipient(RecipientType.TO, new InternetAddress(recipientUsername + "@google.com"));
     email.setSubject(subject);
     email.setText(bodyText);
     return email;
   }
 
+  /**
+   * Create a MimeMessage using the parameters provided.
+   *
+   * @param bodyText body text of the email
+   * @return the MimeMessage to be used to send email
+   * @throws MessagingException if there was a problem accessing the Store
+   */
+  private MimeMessage createEmailWithMultiRecipients(
+      String firstRecipientUsername, String secondRecipientUsername, String bodyText)
+      throws MessagingException {
+
+    Properties props = new Properties();
+    Session session = Session.getDefaultInstance(props, /* authenticator= */ null);
+
+    MimeMessage email = new MimeMessage(session);
+
+    email.setFrom(new InternetAddress(APPLICATION_EMAIL));
+    email.addRecipients(
+        RecipientType.TO,
+        new InternetAddress[] {
+          new InternetAddress(firstRecipientUsername + "@google.com"),
+          new InternetAddress(secondRecipientUsername + "@google.com")
+        });
+    email.setSubject("Ad-lib: Match Found");
+    email.setText(bodyText);
+    return email;
+  }
+
   /** Function that access its api and using it sends an email */
-  //   TODO(#35): Create a dummy email for ad lib itself to send emails.
   //   TODO(#36): Replace body to send real link to user instead of generic.
-  public void notifyUser() throws MessagingException, IOException, GeneralSecurityException {
-    NetHttpTransport HTTP_TRANSPORT = GoogleNetHttpTransport.newTrustedTransport();
+  public void sendMatchEmail(String firstMatchRecipientName, String secondMatchRecipientName)
```

Comment:
Sending a match email to only one person, how about sendMatchEmail(String recipientUsername, String matchUsername). This would be called twice for each participant, and once in each doGet call.

Reply:
i am unclear on as to what you mean, the recipient username would refer to who is going to receive this match email but the match username is going to be who they are matched with ?